tango_simlib/tests/test_tango_sim_generator.py

```diff
@@ -259,7 +260,7 @@ def test_device_attribute_list(self):
         not_added_attr_names = not_added_attr.value
 
         expected_attributes = []
-        for attr_prop in self.sim_file_parser._device_attributes.values():
+        for attr_prop in list(self.sim_file_parser._device_attributes.values()):
```

Comment:
Maybe no list casting needed here either? The tuts on this seems to say that it only be an issue if the 'self.sim_file_parser._device_attributes' dict was being modified in the loop somewhere?

Reply:
@lanceWilliams is correct, no need for list casting as `dict_value`'s are treated as lists when being iterated on.

![image](https://user-images.githubusercontent.com/7910856/71346285-31c85980-2570-11ea-82d6-9ca5edd07d5a.png)

Reply:
I thought so too but look at job 49 and job 50 on jenkins and notice the changes and the failed tests.